queries/bash/highlights.scm

```diff
@@ -0,0 +1,110 @@
+[
+ "("
+ ")"
+ "{"
+ "}"
+ "["
+ "]"
+ ] @punctuation.bracket
+
+[
+ ";"
+ ";;"
+ (heredoc_start)
+ ] @punctuation.delimiter
+
+[
+ ">"
+ "<"
+ "&"
+ "&&"
+ "|"
+ "||"
+ "="
+ "=="
+ "!="
+ ] @operator
+
+[
+ (string) 
+ (raw_string)
+ (heredoc_body)
+] @string
+
+[
+ "if"
+ "then"
+ "else"
+ "elif"
+ "fi"
+ "case"
+ "in"
+ "esac"
+ ] @conditional
+
+[
+ "for"
+ "do"
+ "done"
+ "while"
+ ] @repeat
+
+[
+ "declare"
+ "export"
+ "local"
+ "readonly"
+ "unset"
+ ] @keyword
+
+[
+ (special_variable_name) 
+ ("$" (special_variable_name))
+ ] @constant
+
+((word) @constant
+  (#match? @constant "SIG(INT|TERM|QUIT|TIN|TOU|STP|HUP)"))
+
+((word) @boolean
+  (#match? @boolean "true|false"))
+
+((word) @number
+  (#match? @number "^\d*$"))
+
+(comment) @comment
+(test_operator) @string.
+
+(command_substitution
+  [ "$(" ")" ] @punctuation.bracket)
+
+
+(function_definition
+  name: (word) @function)
+
+(command_name (word)) @function
+
+(command 
+  argument: [
+             (word) @parameter
+             ((word) @number
+              (#match? @number "^\d*$"))
+             (concatenation (word) @parameter)
+             ])
+
+(file_redirect
+  descriptor: (file_descriptor) @operator
+  destination: (word) @parameter)
+
+
+("$" (variable_name)) @identifier
```

Comment:
There is a `(variable_name)` capture below, they seem to overlap

Reply:
Same issue as above the sibling query is necessary for the "$" to considered the same as the "(variable_name)"